src/main/java/com/github/emcat/Emcat.java

```diff
@@ -0,0 +1,49 @@
+package com.github.emcat;
+
+import picocli.CommandLine;
+import picocli.CommandLine.Option;
+import picocli.CommandLine.Command;
+
+import java.util.concurrent.Callable;
+
+import com.github.emcat.metric_calculators.MethodDescriptor;
+import com.github.emcat.metric_calculators.MethodMetricsAggregator;
+import com.github.emcat.metric_calculators.MethodMetricsData;
+
+@Command(name = "emcat", description = "Calculates given NCSS and cyclomatic complexity of a given method.", version = "1.0")
+public class Emcat implements Callable<Integer> {
```

Comment:
Why do you need callable interface here? No  need for it, in your case you can just use a class.